### PR TITLE
chore(vrl): Get the playground up-and-running on https://playground.vrl.dev

### PR DESCRIPTION
**Pull request by jonathanpv**
Closes https://github.com/vectordotdev/vector/issues/14711
Also gets us closer to close out 
https://github.com/vectordotdev/vector/issues/14653

### CODE REVIEW

File: lib/vrl/web-playground/netlify.toml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  # Directory to change to before starting a build.
+  # This is where we will look for package.json/.nvmrc/etc.
+  # If not set, defaults to the root directory.
+  base = "lib/vrl/web-playground/"
+
+  # Directory that contains the deploy-ready HTML files and
+  # assets generated by the build. This is relative to the base
+  # directory if one has been set, or the root directory if
+  # a base has not been set. This sample publishes the directory
+  # located at the absolute path "root/project/build-output"
```
Comment (spencergilbert) [inline lib/vrl/web-playground/netlify.toml:13]: This comment isn't accurate anymore, and we can probably clean them up later.

**Review by spencergilbert — APPROVED** [review]
This doesn't _seem_ to cause problems with the vector.dev site, but next step should definitely be configuring netlify to not build for both sites on every PR. This also isn't _yet_ at https://playground.vrl.dev

**Comment by jszwedko** [comment]
Just a heads up that I added the CNAME for playground.vrl.dev. It appears to be resolving now so should work once the site is deployed.

I might add a banner saying this is a work-in-progress at the top before merging this.

Could go old school and include a gif from https://www.theatlantic.com/technology/archive/2015/10/an-insane-collection-of-1990s-gifs/408799/

**Review by jszwedko — APPROVED** [review]

**Comment by JeanMertz** [comment]
Looks like [playground.vrl.dev](https://playground.vrl.dev/) no longer works:

> Not Found - Request ID: 01GG75BT2N21Q1C6W84QV2RJ4J

**Comment by jszwedko** [comment]
> Looks like [playground.vrl.dev](https://playground.vrl.dev/) no longer works:
> 
> > Not Found - Request ID: 01GG75BT2N21Q1C6W84QV2RJ4J

Yeah, I was talking to Jon about this yesterday. The Netlify deploy failed on `master`. I believe he was going to look into it today.

**Comment by jonathanpv** [comment]
The build failed at getrandom which we can fix by enabling the js feature 
Have this pr to address it
https://github.com/vectordotdev/vector/pull/14949

**Pull request merged**